src/pybind/mgr/dashboard/frontend/src/app/shared/components/alert-panel/alert-panel.component.html

```diff
@@ -38,12 +38,3 @@
 <ng-template #content>
   <ng-content></ng-content>
 </ng-template>
-
-<div class="text-right"
```

Comment:
Not sure the back button should not be shown on error. What was your reason to remove it?

Reply:
Combination of the following:
- Was only used in 1 page, RgwUserForm
- We can achieve the same by clicking the breadcrumb or browser back button
- Would require more code in the directive for it to work
- The button looked a bit lost, alone in the right side of a wide page

I can revert this if needed.

Reply:
Fine with me, thanks for the explanation :)